benchmarking/nursery/benchmark_dehb/requirements.txt

```diff
@@ -1,2 +1,3 @@
 syne-tune[gpsearchers,kde,blackbox-repository,yahpo,aws]
 tqdm
+sortedcontainers  # Remove in version > 0.4.1
```

Comment:
The reason for these temp changes is that syne-tune dependencies are installed from the latest PyPI release. Since `sortedcontainers` is a new dependence, it is not picked up from there.